columbo/_interaction.py

```diff
@@ -419,7 +420,9 @@ def validate(self, value: str, answers: Answers) -> ValidationResponse:
         :return: A ValidationFailure or ValidationSuccess object.
         :raises ValueError: The value for `options` did not have the correct type.
         """
-        options = to_value(self._options, answers, list)
+        options = to_value(
+            list(to_labeled_options(self._options, answers).keys()), answers, list
+        )
```

Comment:
```suggestion
        options = list(to_labeled_options(self._options, answers).keys())
```
I don't think the call to `to_value()` is required. `to_labeled_options()` resolves the dynamic case, then `list(ABC.keys()` ensures that the result is `list[str]`.